src/Pulsar.Client/Internal/HttpLookupService.fs

```diff
@@ -0,0 +1,217 @@
+﻿namespace Pulsar.Client.Internal
+
+open System.IO
+open System.Net.Http
+open System.Text.Json.Serialization
+open Pulsar.Client.Api
+open Pulsar.Client.Common
+open System
+open System.Net
+open FSharp.UMX
+open System.Text.Json
+open Microsoft.Extensions.Logging
+open Pulsar.Client.Schema
+
+type internal HttpLookupService (config: PulsarClientConfiguration, _connectionPool: ConnectionPool) =
+
+    let httpClient = new HttpClient(new SocketsHttpHandler(
+        PooledConnectionLifetime = TimeSpan.FromSeconds(30),
+        AllowAutoRedirect = true
+    ))
+    let jsonOptions = JsonSerializerOptions(
+        PropertyNameCaseInsensitive = true
```

Comment:
I'm not sure this is the best solution. If the fields are lowercase, it's better to just use lower-case strategy than case insensitive or just add [<JsonPropertyName>] attributes over the properties (like in AuthenticationOauth2.fs)

Reply:
In fact these api return fields as lower camel case format. I set  `[<JsonPropertyName>]` in below types:
```
PartitionedTopicMetadata
HttpLookupTopicResult
HttpSchemaInfo
```